packages/wonder-blocks-toolbar/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "name": "@khanacademy/wonder-blocks-toolbar",
+  "version": "0.0.1",
+  "design": "v1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "",
+  "main": "dist/index.js",
+  "source": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@khanacademy/wonder-blocks-core": "^1.0.1"
+  }
```

Comment:
You'll need to include `react` and `aphrodite` as `peerDependencies`.

Reply:
Will do!